codegen/smithy-go-codegen/build.gradle.kts

```diff
@@ -20,6 +20,7 @@ extra["displayName"] = "Smithy :: Go :: Codegen"
 extra["moduleName"] = "software.amazon.smithy.go.codegen"
 
 dependencies {
+	implementation("software.amazon.smithy:smithy-cli:$smithyVersion")
```

Comment:
This should just be in the smithy-go-codegen-test build script.

Reply:
Fixed this up by putting the `smithy-cli` in a `buildScript` section in the correct project.